cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -179,123 +154,103 @@ func main() {
 		os.Exit(1)
 	}
 
-	runnerScaleSetSession, err := actionsServiceClient.CreateMessageSession(ctx, runnerScaleSet.Id, hostName)
-	if err != nil {
+	retries := 3
+	var lastErr error
```

Comment:
This is literally my personal opinion but should we name lastErr as like `messageSessionErr`?
Then, we can group it like:
```go
var (
    messageSessionErr error
    runnerScaleSetSession *github.RunnerScaleSetSession
)
```
To show the scope of this err. Just a suggestion, does feel free to ignore it entirely :smile:

Reply:
Good idea. Done